Dockerfile.arm32v7

```
@@ -17,15 +17,18 @@
 FROM arm32v7/python:3.7.6-buster
 
 # Upgrade all packages to meet security criteria
-RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get upgrade -y && apt-get install sudo && rm -rf /var/lib/apt/lists/*
 
 # arm32v7 is the CPU that is integrated into Raspberry Pi 4
 
-WORKDIR /workspace
-RUN mkdir assets
+RUN useradd --create-home max && echo "max ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+WORKDIR /home/max
+RUN cd /home/max && mkdir assets
```

```suggestion
RUN mkdir assets
```

Also move `USER max` before this line